S/S2Geometry/build_tarballs.jl

```diff
@@ -34,6 +34,7 @@ platforms = [
     Platform("x86_64", "linux"; libc="musl"),
     Platform("aarch64", "linux"; libc="musl"),
     Platform("x86_64", "macos"),
+    Platform("aarch64", "macos"),
```

Comment:
Something like (I haven't counted the number of parentheses, adjust as needed)
```julia
platforms = supported_platforms(; exclude=p -> !((Sys.islinux(p) || Sys.isapple(p)) || nbits(p) == 64))
```
looks more automatic than manually listing the platforms

Reply:
```suggestion
platforms = supported_platforms(; exclude=p -> !((Sys.islinux(p) || Sys.isapple(p)) && nbits(p) == 64))
```